scoreboard.go

```diff
@@ -0,0 +1,124 @@
+package interactor
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+type (
+	Scoreboard struct {
+		EventId     Identifier `json:"event_id"`
+		Time        ApiTime    `json:"time"`
+		ContestTime ApiRelTime `json:"contest_time"`
+		State       State      `json:"state"`
+		Rows        []Row      `json:"rows"`
+	}
+
+	State struct {
+		Started      ApiTime `json:"started"`
+		Ended        ApiTime `json:"ended"`
```

Comment:
These state times can be empty. Then they should be pointers right?

Reply:
I think I misread the table, all of them should be pointers indeed. Thawed also should be omitempty since it is not required.